internal/addon/var.go

```diff
@@ -25,6 +25,16 @@ const (
 	Metrics        Signal = "metrics"
 	Logging        Signal = "logging"
 	Tracing        Signal = "tracing"
+
+	ClfGroup         = "logging.openshift.io"
+	ClfResource      = "clusterlogforwarders"
+	ClfName          = "instance"
+	ClusterLoggingNS = "openshift-logging"
+
+	OtelcolGroup    = "opentelemetry.io"
+	OtelcolResource = "opentelemetrycollectors"
+	OtelcolName     = "spoke-otelcol"
+	OtelcolNS       = "spoke-otelcol"
```

Comment:
The naming of these variables did not do a good job of portraying what Name/Namespace are we referring to here. These are both the name & namespace of the resources that will be created on the spoke clusters. These don't change and are always the same unless we change the manifests. I've added tests so that if we do this the tests fail and we need to update these names

Reply:
hi @JoaoBraveCoding, it seems that the health checking implementation in this PR is static - if a user modifies `ClusterManagementAddOn` e.g. adds a new collector to a spoke or cluster set, the health of that collector is not reflected here. This use case was discussed as part of the proposal https://hackmd.io/aBUzPTEZRuCPp_kZqm4x2A?view

The addon should reflect the health of all collectors specified in the `placements`.

Reply:
So the health checking is indeed static for the moment because the resources MCOA currently generate are also static in their namespace/name (CLF is always called "mcoa-instance", as of this PR, and OTELCol is also called "mcoa-instance", we can change to a different name I'm not hard on that). If the user changes the OTELCol reference on `ClusterManagementAddOn`, there is no problem because MCOA only cares about the `Spec` of the resource.

In the [hackmd](https://hackmd.io/aBUzPTEZRuCPp_kZqm4x2A?view) indeed there is a future requirement that talks about deploying multiple instances of OTELCol (Deployment + Daemonset). However, nowadays this is not possible from the POV of the addon-framework. The addon-framework doesn't support 2 or more references of the same GVK. To change this, we will need to open a PR to the addon-framework to implement this functionality.
The same happens with the Health Probes, currently, they do not allow a dynamic list of `ProbeFields/ResourceIdentifier` this is something that we will need to add support for in the future.

Reply:
It's an important requirement for us to support multiple collectors, I am not sure we can rely on getting this change into the addon framework. 

@iblancasa could you please chime in here and describe your idea with implementing this in a similar way how secrets are handled by the MCOA?

Reply:
@pavolloffay FWIW the ACM ticket we now they are interested for external contributions is [ACM-11509](https://issues.redhat.com/browse/ACM-11509). IMHO we will get either way into a state that our addon will support multiple GVKs. However we still have time till 2.12 to try contributing to the addon-framework first and then fall back if needed to custom code. WDYT?